src/themes/WordpressStandard/Controller/PostController.php

```diff
@@ -35,6 +35,19 @@ public function listAction()
     }
 
     /**
+     * Category list action.
+     *
+     * @return bool|string
+     */
+    public function categoryListAction()
+    {
+        $context = Timber::get_context();
+        $context['posts'] = Timber::get_posts();
+
+        return Timber::render('pages/posts/category-list.twig', $context);
```

Comment:
Standard says that the name must be in snake_case and 'posts' must be in singular.

```php
return Timber::render('pages/post/category_list.twig', $context);
```